### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by sneeu-leeu**
In this milestone I:
- Created a 'person' class
- Created 2 classes that inherits from 'person' : Students & Teachers

### CODE REVIEW

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Invalid submission 

Hi,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has filing linters.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by sneeu-leeu** [comment]
Thanks @arslanbisharat, it was a simple linter that did not show up locally and I was eager to submit.

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_accsessor :name, :age
```
Comment (DeliceLydia) [inline person.rb:2]: # Optional
- [ ] Remember to correct this to ` attr_accessor`

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @sneeu-leeu, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**